src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.csproj

```diff
@@ -89,19 +89,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="build\common\MSTest.TestAdapter.props" Condition=" '$(TargetFramework)' != '$(UwpMinimum)' ">
+    <None Update="build\common\*" Condition=" '$(TargetFramework)' != '$(UwpMinimum)' ">
```

Comment:
Is this a change in behavior? Previously the props was done with a condition while the targets didn't have a condition.

Reply:
Some "bugs" + dead code